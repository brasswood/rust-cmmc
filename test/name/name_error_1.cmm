# How many errors does the Oracle report?
int a;
int a;
int a;
int b;
int b;
