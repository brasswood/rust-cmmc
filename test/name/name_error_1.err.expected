FATAL [3,5]: Multiply declared identifier
FATAL [4,5]: Multiply declared identifier
FATAL [6,5]: Multiply declared identifier
Name Analysis Failed
